cid/common.py

```diff
@@ -483,10 +478,12 @@ def create_datasets(self, _datasets: list) -> dict:
                 print(f'\tfound: {k}', end='')
                 if len(v.keys()) > 1:
                     # Multiple datasets
-                    selected = questionary.select(
-                        f'Multiple "{k}" datasets detected, please select one',
-                        choices=v.keys()
-                    ).ask()
+                    print()
```

Comment:
Why it is here?

Reply:
indeed needs to be remvoed, as this print is moved to the get_parameters function. was initally due to end="" in prints